### PR TITLE
fix misleading help message: actually default loss function is "softmax", not "negative sampling"

### PR DESCRIPTION
**Pull request by pliable-statistics**

### CODE REVIEW

**Comment by cpuhrsch** [comment]
A fix like this was landed separately. Thank you for the request @pliable-statistics!
